frontend/.env

```diff
@@ -2,3 +2,4 @@ REACT_APP_TITLE=SimpleReport
 REACT_APP_DESCRIPTION=SimpleReport Test Management Prototype Application
 REACT_APP_ICON=usds-32x32.png
 REACT_APP_BACKEND_URL=https://prime-data-input-sandbox-backend.app.cloud.gov/graphql
+REACT_APP_OKTA_URL=https://simplereport.cdc.gov/
```

Comment:
Would suggest either "LOGIN_PAGE" or "HOME_PAGE" for this rather than OKTA_URL, in that it is not in fact an Okta URL. 😃